src/components/DailyVideo.tsx

```diff
@@ -167,13 +167,13 @@ export const DailyVideo = forwardRef<HTMLVideoElement, Props>(
         if (!onResize || !video) return;
 
         let frame: ReturnType<typeof requestAnimationFrame>;
-        const handleResize = () => {
-          if (!video) return;
```

Comment:
I believe `video` would have always been defined at this point. Instead, I added another check for `videoEl.current` further down.